content/metadata.yaml

```diff
@@ -40,3 +40,13 @@ authors:
       string: "None"
       lastapproved: !!str 2020-03-22
     funders: the Gordon and Betty Moore Foundation (GBMF 4552)
+  -
+    github: mprobson
+    name: Michael P. Robson
+    initials: MPR
+    orcid: 0000-0002-4859-0033
+    email: michael.robson@villanova.edu
+    contributions:
+      - Document Cleanup
```

Comment:
Can you use one of the CRediT taxonomy roles? I'm imagining this is "Methology" "Validation" or "Software". @rando2 what do you think is the appropriate one?

These are the standard roles: https://casrai.org/credit/